templates/bot/js/default/dialogs/mainDialog.js

```diff
@@ -58,6 +59,16 @@ class MainDialog extends RootDialog {
     return await stepContext.beginDialog(TEAMS_SSO_PROMPT_ID);
   }
 
+  async dedupStep(stepContext) {
+    const tokenResponse = stepContext.result;
+    if (tokenResponse) {
```

Comment:
please add some comments to help users understand the logic

Reply:
Sure